### DISCUSSION TITLE
CSS3中使元素居中的各种方式 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#77] [General]
# CSS3中使元素居中的各种方式 - 夏沫花火zzz🌙 (Muska_Ami)のLife

Keep doing, keep loving❤.

https://blog.1l1.icu/2024/06/29/css3-zhong-shi-yuan-su-ju-zhong-de-ge-chong-fang-shi/

**Comment by Muska-Ami** [comment]
flex部分，`flex-direction` 应该是设置子元素排列方向
